Documentation/Logo.md

```diff
@@ -5,28 +5,31 @@ By default, there is _not_ one included in the firmware. This means that once fl
+There are community logo's already converted and ready to use in [IronOS-Meta\releases](https://github.com/Ralim/IronOS-Meta/releases).
+Download the zip for Pinecil or Miniware and then install using the Flashing instructions below.
 
-There are community logo's already converted available for download in the releases in [IronOS-Meta](https://github.com/Ralim/IronOS-Meta/).
+If you want to make custom art then it needa to be converted with a Python script. The script and other needed files are in [IronOS-Meta](https://github.com/Ralim/IronOS-Meta/). Go to that folder, then it is easiest to select the green Code button (upper right), then Download Zip. This way you get all the files you need and some extras. You only need what is inside Boot Logos. Put your custom image inside the Boot Logos folder with all python script files already there.
+
+The Python script converts an image passed into it on the command line into both a `.hex` file and a `.dfu` to be uploaded to the iron in DFU mode. The image can be in color and any size, but it will be resized and converted to 1-bit color. However, it looks best if you create a 96x16 image (Png or Bmp) in any image editor and color the pixels black & white manually.
 
 The converter requires at least Python 3 and Pillow (if you don't have it, it will tell you to install PIL, which is an old version of the same thing). See [this page](https://stackoverflow.com/a/20061019/6705343) on StackOverflow about installing it.
 
 Follow online instructions for installing Python and Pillow. For Windows, it is recommended to use Windows PowerShell instead of Command.
-Open Powershell (run as administrator), type python to install it, it will open microsoft store where you can install it.
+Open Powershell (run as administrator), type python to install it, it will open microsoft store where you can install it free.
 
-Go back to Powershell, what works can vary, but this command may work:
+Go back to Powershell and install Pillow. What works can vary, but this command may work:
 
     python -m pip install Pillow
 or 
     python3 -m pip install pillow
 
-Then, to convert an image:
+Then, to convert a custom image (change infile.png to the name of your image):
 
-- `python img2logo.py infile.png out -m` for Miniware
-- `python img2logo.py infile.png out -p` for Pinecil
+- `python img2logo.py infile.png out -m`   for Miniware
+- `python img2logo.py infile.png out -p`   for Pinecil
 
-Run `python img2logo.py --help` to see available options. Replace word python with python3 if you have multiple versions of python installed.
-Make sure your image file is in the same folder as scripts img2logo.py, output_dfu.py, and output_hex.py.
+Run `python img2logo.py --help` to see available options. Replace the word python with python3 if you have multiple versions of python installed. 
+_Note Bene_: make sure your image file is in the same folder as script files (img2logo.py, output_dfu.py, output_hex.py).

```

Comment:
Isn't this `Nota Bene`.
Though it might be more translatable across languages as just "Note:"

Reply:
it is latin like saying Quid Pro Quo which is also latin.   so it's commonly used just like  Quid Pro quo  but I can change it to simple Note word.   Note Bene  means important note.

Reply:
oh I see I misspelled it.  you are right I meant to type Nota Bene  that was a typo.  but I just changed it  to be simply Note:

Reply:
also I figured out how to change my Default from master to dev  for document updates.